internal/config/yaml_parser_test.go

```diff
@@ -34,6 +34,9 @@ func validateDefaultConfig(mountConfig *MountConfig) {
 	AssertEq("INFO", mountConfig.LogConfig.Severity)
 	AssertEq("", mountConfig.LogConfig.Format)
 	AssertEq("", mountConfig.LogConfig.FilePath)
+	AssertEq(512, mountConfig.LogConfig.LogRotateConfig.MaxFileSizeMB)
```

Comment:
Thank you for the input. I will keep this in mind for any future changes. I think for all the current changes, we would like to fail the unit test, if the values are incorrect. Please let me know if you think otherwise :)